### ISSUE TITLE
Create dirty JSON test data

### ISSUE THREAD

**Issue by DianeWold** [cdisc-org/ddf-core-poc#157]
This will be done by editing the "clean" test data, then making changes to trigger existing created rules.

**Comment by DianeWold** [comment]
Added errors that should trigger the following rules:
DDF00005 added a second sponsor identifier
DDF00006 removed the lower bound on a window
DDF00025 added a window description to an anchor
DDF00008 scheduled instance with both a default condition and an exit
DDF00013 required BC property not enabled
DDF00015 phase not in terminology
DDF00020 other study amendment without specified reason
DDF00036 anchor not start to start
DDF00012 no main timeline
DDF00007 anchor with different relative to and relative from links
DDF00031 non-anchor with relative to and relative from links the same
DDF00033 duplicate study amendment reasons
DDF00022 child link to itself
DDF00021 three errors, for previous, next, and defaultcondition links to itself
DDF00014 two timings with the same label
DDF00034 no response code enabled

I did not add errors for the following rules
DDF00008 couldn't find an administration duration will vary in the json file
DDF00011 complement of DDF00012 couldn't have both no main timeline and two main timelines

**Comment by DianeWold** [comment]
At one point, my original data file wouldn't save so I made a copy and renamed it
Both are here https://cdisc.sharepoint.com/sites/ddf/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2Fddf%2FShared%20Documents%2FDDF3a%2FJsonTestFiles&viewid=9a467a7f%2D5356%2D4249%2D808a%2D07dba21b0b7d
I don't seem to be able to save the new copy. Should I delete the old one to make space?

**Comment by DianeWold** [comment]
Issues with saving the dirty json file resolved.  The correct file is the one with "dirty1" in its name.

**Comment by ASL-rmarshall** [comment]
I have updated the [CDISC_Pilot_Study_dirty1.json](https://cdisc.sharepoint.com/sites/ddf/_layouts/15/download.aspx?UniqueId=7a70a7baffdb46999f420f16e1ac99a4&e=doXDSX) file in SharePoint with the changes I made to get the command line interface (CLI) to work correctly.  Apart from the formatting changes (adding/removing line breaks according to the usual JSON formatting for CDISC repositories - i.e. as applied by the `prettier` JSON formatter), these are the changes I made:
- Changed `StudyDesign.blindingSchema` from an `AliasCode` to a normal `Code`.
- Renamed `StudyDesign.population` to `StudyDesign.populations` and made its contents a list.
- Change the format used for `parameterMap` in both dictionaries, e.g., changed from:
    ```json
                "parameterMap": {
                  "min_age": {
                    "klass": "StudyDesignPopulation",
                    "id": "StudyDesignPopulation_1",
                    "attribute": "plannedMinimumAge"
                  },
    ```
    to:
    ```json
                "parameterMap": {
                  "min_age": "<usdm:ref klass=\"StudyDesignPopulation\" id=\"StudyDesignPopulation_1\" attribute=\"plannedMinimumAge\"></usdm:ref>",
    ```
